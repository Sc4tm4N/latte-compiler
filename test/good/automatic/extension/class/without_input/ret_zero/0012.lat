int calculate(int x) {
  return x + x * x;
}

class Node {
  int a;

  int calculate(int x) {
    return a + x * x;
  }

  int checker(int g) {
    printInt(self.calculate(g));
    printInt(calculate(g));
    return self.calculate(g) + calculate(g);
  }
}

int main() {
  Node node = new Node;
  node.a = 100;
  printInt(node.checker(7));
  return 0;
}
