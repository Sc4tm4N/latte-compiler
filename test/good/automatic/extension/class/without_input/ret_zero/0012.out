149
56
205
